Client/Frontend/Browser/TabManager.swift

```diff
@@ -670,16 +672,12 @@ extension TabManager {
         let archiver = NSKeyedArchiver(forWritingWith: tabStateData)
         archiver.encode(savedTabs, forKey: "tabs")
         archiver.finishEncoding()
-        tabStateData.write(toFile: path, atomically: true)
+        let status = tabStateData.write(toFile: path, atomically: true)
+        print(status)
     }
 
     func preserveTabs() {
-        // This is wrapped in an Objective-C @try/@catch handler because NSKeyedArchiver may throw exceptions which Swift cannot handle
```

Comment:
Yea. Thats what I was trying to understand. How could NSKeyedArchiver actually crash? The one you pointed me to was a weird dealloc case. That seems like something that could be caused by our try/catch hack? How often did that crash happen?